sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -372,12 +372,48 @@ class HiveCatalogedDDLSuite extends DDLSuite with TestHiveSingleton with BeforeA
       assert(table.provider == Some("org.apache.spark.sql.hive.orc"))
     }
   }
+
+  test("Database Ownership") {
+    val catalog = spark.sessionState.catalog
+    val databaseNames = Seq("db1", "`database`")
```

Comment:
why do we need to test name with backticks? How is it related to owner?

Reply:
they are not related,  this test is only to test Hive DDLs. I will make it simpler